src/App.tsx

```diff
@@ -14,11 +14,100 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[];
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: goodsFromServer,
+    selectedGoods: ['Jam'],
+  };
+
+  addWord = (word: string) => {
+    this.setState((prevState) => ({
+      selectedGoods: [...prevState.selectedGoods, word],
+    }));
+  };
```

Comment:
This method always adds a word, even if it already exists in the array, you need to create a condition that will check if the array has this word.